query/src/lib.rs

```diff
@@ -129,6 +129,15 @@ impl PredicateFn {
 }
 
 #[derive(Clone, Debug, Eq, PartialEq)]
+pub enum Direction {
+    Ascending,
+    Descending,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Order(pub Direction, pub Variable);   // Future: Element instead of Variable?
```

Comment:
We could do it by allowing you to name aggregate expressions, as SQL allows:

```
[:find [(max ?age) ?oldest]
 :where [_ :foo/age ?age] …
 :order ?oldest]
```

but at present there's no way to do it.